src/plugins/ResizeReorder/ResizerKnob.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ResizerKnob
+ * @typechecks
+ */
+
+import React from 'react';
+import cx from 'cx';
+import joinClasses from 'joinClasses';
+import FixedDataTableEventHelper from 'FixedDataTableEventHelper';
+import ResizerLine from 'ResizerLine';
+import clamp from 'clamp';
+import DOMMouseMoveTracker from 'DOMMouseMoveTracker';
+
+
+class ResizerKnob extends React.Component {
+
+  initialState = {
```

Comment:
Could you simplify `initialState` a bit more?

Do we need to store the `instance` ref to state?

Do we need to store all of these properties: `mouseXCoordinate`, `initialMouseEvent`, and `totalDisplacement` ? 
I feel like some of these cane be instead derived. 
Also could you put a small inline comment here explaining what these are?

Reply:
initialMouseCoordinated -> Required to calculate displacement during resizing from initial position.
totalDisplacement -> Used to calculate newWidth during onMouseUp
MouseXCoordinate -> Not required. (Will remove it)